gooddata.gemspec

```diff
@@ -6,6 +6,9 @@ Gem::Specification.new do |s|
   s.name = 'gooddata'
   s.version = GoodData::VERSION
   s.licenses = ['BSD']
+  if RUBY_PLATFORM =~ /java/
```

Comment:
Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.